src/bot/commands/report.ts

```
@@ -22,10 +23,10 @@ bot.onText(/\/report (.+)/, (msg, match) => {
       `🆔 <b>User ID:</b> <code>${msg.from?.id}</code>\n\n` +
       `📝 <b>Report:</b>\n<i>${reportMessage}</i>`;
 
-   // Forward to personal chat
-   bot.sendMessage(ADMIN_ID || 1329532701, formattedMessage, {
-      parse_mode: "HTML",
-   }).catch((err) => console.error("Error forwarding report:", err));
+    // Forward to personal chat
```

ESLint

[error] Delete `··`

(prettier/prettier)